### ISSUE TITLE
It would appear this library has been replaced by https://github.com/parse-community/node-apn , which is identical but works

### ISSUE THREAD

**Issue by smhk** [node-apn/node-apn#739]
This library appears to be totally dead and doesn't work at all.   

Should someone shut it down ??

```
] npm uninstall apn
] npm install @parse/node-apn
```

Everything about "@parse/node-apn" appears to be identical

- you do not have to **change any code, it is identical**
- the instructions/doco on this web site and that are identical
- only difference is the new one works, it seems!  :O 

Hope it helps. Surely this github should be closed?

**Comment by auldo** [comment]
I agree with this. E.g., shutdown() function is not working properly with this package, s.t. sendNotification calls don't terminate. Switched to the package you linked and it works :)
